spec/cache_spec.rb

```diff
@@ -1,14 +1,25 @@
 require 'spec_helper'
-require 'gris/identity'
 
-describe 'Gris.cache' do
-  before(:each) do
-    @cache = Gris.cache
+describe 'Gris::Caching' do
```

Comment:
This looks like a accidental overwriting of the Gris.cache specs (minimal such as they were). We'll likely want to restore those to something like - 

```ruby
require 'spec_helper'

describe 'Gris.cache' do
  let(:cache) { Gris.cache }

  it 'returns nil by default' do
    expect(cache.read('x')).to be_nil
  end

  it 'permits writing to and retrieving from cache' do
    cache.write('x', 'abc')
    expect(cache.read('x')).to eq('abc')
  end
end
```